pygithub/im_commits_pygithub.py

```diff
@@ -0,0 +1,66 @@
+"""Demo designed to find and store information about PyGithub."""
+from github import Github
+from pydriller import RepositoryMining, GitRepository
+from pydriller.domain.commit import ModificationType
+from fastapi import FastAPI
+
+test_dict = {}
+def output_hash_map(dictionary):
+    """Method to print out the hash map."""
+    # print headings
+    print("Branch", "\t Author", "\t Time", "\t Files", "\t Message")
+    # prints hashmap content
+    # for key in dictionary:
+
+
+    for key, _ in dictionary.items():
+        print(key)
+        for items in dictionary[key]:
```

Comment:
But from this line, since like what you actually wanted is just the value of the key, if that's the case, you can just use something like `for value in dictionary.values()`